makefile

```diff
@@ -0,0 +1,8 @@
+VERSION := $(shell git describe --tags)
+# There are the default architecture values can be changed via 'make OS=linux ARCH=amd64'
+OS = darwin
```

Comment:
Maybe we can remove them, and not specify the OS / ARCH for the all build so you can just use make and build the binary for your current platform.
Moreover we could add a release rule in the makefile which could build for each targets we support with borg.
Do not forget to add the -w linker flag in order to remove the DWARF debug symbols from the releases binaries.
And could you add a call to upx for each binary you generate in order to make them smaller.